### ISSUE TITLE
Let's make it portable to other platforms!

### ISSUE THREAD

**Issue by elderkeltir** [lanza/lldb-vscode#29]
Hello,

I have a great idea! can you pls add couple config parameters to allow users of non-macos to put there path to bin directory(where lldb* binaries stored) and what else is needed(python with some modules installed? we can use another config string parameter as path to python).
thus you don't need to build/store anything in the repo. Users of linux or FreeBSD(like I am) could make use of your extension via system installed llvm/python.

seems like win-win and changes are minimal!
just pls don't disable it for freebsd(that if clause you have for os checking doesn't have freebsd listed).
I would try  do this by myself, but code-oss on freebsd doesn't let me to launch my extension hello world. Not sure it's possible from FreeBSD.

So to sum up: few config params to chose needed directories(better files), and list of python modules necessary. Also remove check for macos, just make path to your binaries as default, thus macos users won't even notice changes.

Also you can state in description: officially we support only macos, anything else is up to users, no warranty! =)

Regards,

**Comment by lanza** [comment]
This is out of date, I suggest using code-lldb instead.
